diff --git a/lib/index.js b/lib/index.js
index 4a7eae590bd6f9c6a97588c75f1c5fe7e570c7c7..c891530289832d9819b3b1f033f47b415f41fa33 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -133,27 +133,14 @@ import {toHast} from 'mdast-util-to-hast'
  * @returns {TransformBridge | TransformMutate}
  *   Transform.
  */
-export default function remarkRehype(destination, options) {
-  if (destination && 'run' in destination) {
-    /**
-     * @type {TransformBridge}
-     */
-    return async function (tree, file) {
-      // Cast because root in -> root out.
-      const hastTree = /** @type {HastRoot} */ (
-        toHast(tree, {file, ...options})
-      )
-      await destination.run(hastTree, file)
-    }
-  }
-
+export default function remarkRehype(options) {
   /**
    * @type {TransformMutate}
    */
   return function (tree, file) {
     // Cast because root in -> root out.
     return /** @type {HastRoot} */ (
-      toHast(tree, {file, ...(options || destination)})
+      toHast(tree, {file, ...(options)})
     )
   }
 }
